### DISCUSSION TITLE
Local network / Offline File Sharing? Like PirateBox?

### DISCUSSION THREAD

**Discussion by michaeltoohig** [rejetto/hfs#408] [Q&A]
Can this project be used like PirateBox on a Raspberry Pi? I just want users on a local only network be redirected to a home page where they can download files. This project may be the right one I'm looking for but curious if anyone has used this project in this sort of way as I'm not interested in https or login credentials for users etc.

**Comment by rejetto** [comment]
i'm not sure if it fits your purpose, never heard about piratebox before, but if you have specific questions i'll try to answer

**Reply by michaeltoohig** [reply]
I'm sorry piratebox is this now seemingly defunct project found here https://piratebox.cc/

It was a RaspberryPi with captive portal that allowed anyone to connect to the Wifi network to view and download files or even upload their own for others to later come by and find. I want to do something similar at a local club to allow anyone to download files when they visit but people are not tech literate so it will have to be as simple as "connect to wifi" -> "select files to download" so no extra apps to install or domains to type in to the browser plus it must work in local only network.

I finally got a copy of the project so I see it now (slow internet). Seems pretty close to what I need minus the captive portal. Sorry for the vague initial message.
